pom.xml

```diff
@@ -543,14 +546,39 @@
     </build>
 
     <profiles>
+        <!--
+            The failsafe plugin will execute any tests that start with IT or ends with IT or ITCase
+            These are much slower to execute than unit tests, so they are only executed with the given profile
+            when full testing is desired.  Integration Tests should follow this convention
+        -->
+        <profile>
+            <id>integration-tests</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <version>3.1.2</version>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
         <profile>
             <id>release</id>
             <build>
                 <plugins>
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-source-plugin</artifactId>
-                        <version>3.3.0</version>
+                        <version>3.2.1</version>
```

Comment:
Not really sure of the impact of this, but my IDE displayed errors with the 3.3.0 version, maybe some sort of incompatibility?  We can exclude this change if necessary.